desc/backend.py

```diff
@@ -58,15 +66,15 @@
     )
 )
 
-if use_jax:
+if use_jax:  # noqa: C901 - FIXME: simplify this, define globally and then assign?
```

Comment:
that error code is too much branching logic?

Reply:
Basically, it refers to [cyclomatic complexity](https://en.m.wikipedia.org/wiki/Cyclomatic_complexity), or how many different paths there are through a function